keps/sig-release/1498-kubernetes-yearly-support-period/README.md

```diff
@@ -207,17 +207,17 @@ Unchanged from existing Kubernetes project deprecation criteria.  Deprecations f
 Incremental change elongating existing Kubernetes project version skew policy, i.e.: where “N+/-1” to “N+/-2” and where “N+/-2” to “N+/-3”.
 This matters, especially for external clients programmatically interacting with a long-lived cluster, beyond the cluster control plane components, moved through the upgrade process.
 
-## Implementation History
+Note: The above assumes no change in the number of releases made per year.  If the number of releases per year increases or decreases, that would additional modify the version skew strategy.
 
-Major milestones in the life cycle of a KEP should be tracked in `Implementation History`.
-Major milestones might include
+## Implementation History
 
-- the `Summary` and `Motivation` sections being merged signaling SIG acceptance
-- the `Proposal` section being merged signaling agreement on a proposed design
-- the date implementation started
-- the first Kubernetes release where an initial version of the KEP was available
-- the version of Kubernetes where the KEP graduated to general availability
-- when the KEP was retired or superseded
+- 1.19 is planned to be the first release to officially receive a year of
+  patch support
+- discussion is underway (April/May 2020) between SIG Release, WG LTS, and
+  the broader community (see [k-dev
+  list](https://groups.google.com/forum/#!forum/kubernetes-dev)) to take the circumstances and
```

Comment:
Let's explicitly link the discussions:
- 1.19 schedule: https://groups.google.com/d/topic/kubernetes-dev/IVpiIOZ4WcM/discussion
- annual support to `implementable`: https://groups.google.com/d/topic/kubernetes-dev/tOe2UFB_weQ/discussion

Reply:
Added.